frontend/src/components/SamplesList.tsx

```diff
@@ -186,33 +235,59 @@ export default function SamplesList({
       return [...changes];
     });
 
-    // validate Cost Center inputs
-    if (fieldName === "costCenter") {
-      if (!isValidCostCenter(newValue)) {
-        setAlertContent(COST_CENTER_VALIDATION_ALERT);
-        setShowAlertModal(true);
-      } else {
-        resetAlertIfCostCentersAreAllValid(changes);
```

Comment:
Do we still need this behavior or not with the ways things have been refactored?

Reply:
Good question -- We do. This behavior is still happening. I simplified the React.js states for displaying alert modals, so the new code is shorter but is producing the same behavior.